src/Microsoft.AspNetCore.Mvc.Core/DependencyInjection/MvcCoreServiceCollectionExtensions.cs

```diff
@@ -199,10 +199,11 @@ internal static void AddMvcCoreServices(IServiceCollection services)
                 ServiceDescriptor.Singleton<IFilterProvider, DefaultFilterProvider>());
 
             //
-            // RequestSizeLimit filters
+            // Request body limit filters
             //
             services.TryAddTransient<RequestSizeLimitFilter>();
             services.TryAddTransient<DisableRequestSizeLimitFilter>();
```

Comment:
I do not have a corresponding 'disable' filter for this attribute and would like to know your opinion. I feel we could just use one filter to override any inherited behavior. 
For example, a controller has limits set on the form, but the action can just decorate the attribute (with no explicit limits set) and the default limits take over.
I feel we could have done something similar for RequestSizeLimit filter too (but its too late as it would be a breaking change)

Reply:
Seems reasonable. Maybe we could call this out in the doc comments stating adding a parameterless instance would reset the limits